committeeoversightapp/management/commands/import_data.py

```diff
@@ -0,0 +1,210 @@
+            reader = csv.DictReader(csvfile)
+
+            for row in reader:
+                committee_key = row['Committees']
+                committee_name = row['b']
+                classification = "committee"
+
+                try:
+                    organization = Organization.objects.filter(Q(name__icontains=committee_name, parent=senate, classification=classification) |
+                                Q(other_names__name__icontains=committee_name, parent=senate, classification=classification))[0]
+
+                    new_committee, _ = Committee.objects.get_or_create(lugar_id=committee_key, lugar_name=committee_name, organization=organization)
+
+                except IndexError:
+                    new_committee, _ = Committee.objects.get_or_create(lugar_id=committee_key, lugar_name=committee_name)
+                    bad_rows.append("Not recognized: " + committee_key + ", " + committee_name)
+
+    def add_house_hearings(self):
+        with open('data/final/house.csv', 'r') as csvfile:
+            reader = csv.DictReader(csvfile)
+            row_count = 2
+
+            for row in reader:
+
+                source = row['source']
+                start_date = row['Date']
+                name = row['Hearing/Report']
+                classification = row['Type']
+                committee1 = row['Committee1']
+                committee2 = row['Committee2']
+                subcommittee1 = row['Subcommittee']
+                subcommittee2 = row['Subcommittee2']
+                category = row['Category1']
+
+                if name:
+                    # get or create hearing
+                    event, created = Event.objects.get_or_create(jurisdiction_id = jurisdiction_id,
+                                    name = name,
+                                    start_date = start_date,
+                                    classification = classification)
+
+                    # save committee1 data
+                    if committee1 and not subcommittee1:
+                        new_committee(committee1, event, row_count)
+                    elif subcommittee1 == (committee1 + str(0)):
+                        new_committee(committee1, event, row_count)
+                    elif committee1:
+                        new_committee(subcommittee1, event, row_count)
+
+                    # save committee2 data
+                    if committee2 and not subcommittee2:
+                        new_committee(committee2, event, row_count)
+                    elif committee2 and subcommittee2 == (committee2 + str(0)):
+                        new_committee(committee2, event, row_count)
+                    elif committee2:
+                        new_committee(subcommittee2, event, row_count)
+
+                    if created:
+
+                        # save event source
+                        source = EventSource(event=event, note="spreadsheet", url=source)
+                        source.save()
```

Comment:
You can simply call `create`, and then you do not need to build the object and save it in distinct steps. You can also start to remove some of this obvious in-line comments., e.g. "save event source" - the code speaks for itself.